src/main/settings.py

```diff
@@ -36,6 +36,10 @@
 
 ALLOWED_HOSTS = ["affils.clinicalgenome.org", "affils-test.clinicalgenome.org"]
 
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:3001",
```

Comment:
Wouldn't we need to add `https://curation.clinicalgenome.org` and `https://curation-test.clinicalgenome.org`?

Reply:
You are absolutely correct, my apologies for missing that! I have made this change. I also realized that the API endpoints are completely open, so I added a login required and updated tests.py to validate.